client/src/components/App.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+class App extends React.Component {
+    constructor(props) {
```

Comment:
use 2 space for indenting

Reply:
updated to airBnB linter so all the indenting should be fixed!